src/engine/renderer/tr_shader.cpp

```diff
@@ -1787,6 +1786,11 @@ static void ParseLightFalloffImage( shaderStage_t *stage, const char **text )
 	}
 }
 
+static bool HasNormalFormat( shaderStage_t *stage )
+{
+	return !stage->normalFormat[ 0 ];
```

Comment:
I think with a float we should be explicit, `!= 0`

Reply:
I turned `normalFormat` into an array of `short` instead of `float` as it turns out we don't need floats there, all we store is `{-1, 0, 1}`.